modules/ROOT/pages/configure-message-flows.adoc

```diff
@@ -93,13 +66,43 @@ This is the target message type and the Endpoint where you want to route the tra
 
 * xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
 
-. After you create the source, map, and target sections in the interface, confirm that each shows a green check mark next to it in the flow), and click *Deploy* to deploy the configurations to your Mule Runtime.
+. After you create the source, map, and target sections in the interface, confirm that each shows a green check mark next to it in the flow, and click *Deploy* to deploy the configurations to your Mule Runtime.
 +
-Your message flow configuration is deployed to your Mule runtime engine and you can start receiving messages from your partner. Use the Activity page to track processed messages.
+. Your message flow configuration is deployed to your Mule runtime engine and you can start receiving messages from your partner.
 +
-Once a message flow is successfully deployed, you can track its processed messages from the Activity page.
+. Once a message flow is successfully deployed, you can track its processed messages from the Activity page.
+
+== Test Message Flows
+
+There are a couple of reasons why you should send test messages through the message flow after you deploy:
+
+* Ensure that you are receiving messages and any acknowledgments from your partners.
+
+* Make sure that the DataWeave map is working and your backend systems receive the transformed message.
+
+You can test the flow by deploying it in a sandbox environment, then undeploying it, and finally deploying it to a production environment.
+
+Test your message flow in your environment:
```

Comment:
Clarify: is this the sandbox text or a new text in "your" (production) environment?

Reply:
At one point, I had put in there that you can test in either environment. Many apps like to use the sandbox for testing, but some users will only test in production. A reviewer wanted the environments out because he said it was confusing. I will check with the PM for clarity.

Reply:
Adding the both option back in.